packages/website/ts/globals.d.ts

```diff
@@ -132,21 +133,26 @@ declare class Subprovider {}
 declare module 'web3-provider-engine/subproviders/subprovider' {
     export = Subprovider;
 }
-
-// tslint:disable-next-line:max-classes-per-file
-declare class RpcSubprovider {
-    constructor(options: {rpcUrl: string});
-    public handleRequest(payload: any, next: any, end: (err?: Error, data?: any) =>  void): void;
-}
 declare module 'web3-provider-engine/subproviders/rpc' {
+    import * as Web3 from 'web3';
+    class RpcSubprovider {
+        constructor(options: {rpcUrl: string});
+        public handleRequest(
+            payload: Web3.JSONRPCRequestPayload, next: () => void, end: (err: Error|null, data?: any) =>  void,
+        ): void;
+    }
     export = RpcSubprovider;
 }
-// tslint:disable-next-line:max-classes-per-file
-declare class HookedWalletSubprovider {
-    constructor(wallet: any);
-}
-declare module 'web3-provider-engine/subproviders/hooked-wallet' {
-    export = HookedWalletSubprovider;
+declare module 'web3-provider-engine' {
```

Comment:
We have it in at least two places in code. Maybe it's time for a separate typings package?

Reply:
Will move after your PR get merged ;)